src/components/ThemeProvider/__stories__/ThemeProvider.stories.helpers.tsx

```diff
@@ -201,3 +201,48 @@ export const UsageGuidelinesThemeProvider = () => (
     ]}
   />
 );
+
+export const DescriptionWithLinkMondaySdkIntegration = () => (
+  <>
+    In case you are developing an external application for monday.com. You can use ThemeProvider in combination with the{" "}
+    <Link
+      href="https://developer.monday.com/apps/docs/mondayget#sample-context-objects-for-each-feature-type"
```

Comment:
this should be a link to the Vibe intro page in the SDK docs once ready

Reply:
Shlomi commented it in another pr (https://github.com/mondaycom/monday-ui-react-core/pull/1869/files) and it is also relevant here (maybe even more). I think it is a bit confusing that we call the system theme "system theme" but in the SDK API it called just "theme". maybe we need to align it?
@SergeyRoyt What do you think?

Reply:
Answered there. Let's talk about it tomorrow